controller.rst

```diff
@@ -176,6 +176,9 @@ and ``redirect()`` methods::
         // redirects to a route with parameters
         return $this->redirectToRoute('blog_show', array('slug' => 'my-page'));
 
+        // redirects to a route and mantains the original query string parameters
+        return $this->redirectToRoute('blog_show', $request->query->all());
```

Comment:
This change looks good! I'm just thinking, would it be worth it having the base controller support the following syntax?

```php
return $this->redirectToRoute('blog_show', $request->query);
```

Reply:
In my opinion it's better to not add a new way to get the query string params. Let's keep things simple and minimize the number of methods/features to learn. Thanks!